commerce-configuration-1/.generated/actions/app-management/set-configuration.js

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2025 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+// This file is auto-generated by the pre-app-build hook
+// Do not modify this file directly
+
+import { init } from '@adobe/aio-commerce-lib-config';
+import { 
+  badRequest, 
+  internalServerError, 
+  ok 
+} from '@adobe/aio-commerce-sdk/core/responses';
+
+/**
+ * Get the configuration.
+ * @param params - The input parameters.
+ * @returns The response object containing the updated configuration values.
+ */
+export async function main(params) {
+  try {
+    const lib = init();
+    let body;
+
+    if (params && typeof params === 'object' && Object.keys(params).length === 0) {
+      body = {};
+    }
+
+    const id = params.id
+    const code = params.code
+    const level = params.level;
+    if (!id && !(code && level)) {
+      return badRequest({ 
+        body: {
+          code: 'INVALID_PARAMS',
+          message: 'Either id or both code and level query params are required'
+        } 
+      });
+    }
+
+    const candidateConfig = params.config ?? body?.config;
+    if (!candidateConfig || !Array.isArray(candidateConfig)) {
+      return badRequest({ 
+        body: {
+          code: 'INVALID_BODY',
+          message: 'request must include a config array in params.config or body.config'
+        } 
+      });
+    }
+
+    const payload = { config: candidateConfig };
+
+    const libAny = lib;
+    const result = id
+        ? await libAny.setConfiguration(payload, id)
+        : await libAny.setConfiguration(payload, code, level);
```

Comment:
I know it's not related to changes in this particular PR.
Similar to my comment about `getConfiguration`, it is not clear what arguments should be provided to the method. Public API can be misleading for the end user, as the same method can have different behavior.
It should be two or three different methods:
```
setConfigurationById(id, payload);
setConfigurationByCode(code, payload);
setConfigurationByCodeAndLevel(code, level, payload);
```